project_management/workshop_advert.md

```diff
@@ -0,0 +1,56 @@
+
+This workshop is for people who are:
+*	Interested in reproducibility, containers, Docker or continuous integration; 
+*	Already familiar with R Markdown or Jupyter notebooks;
+*	Happy to support others during practical sessions!
+
+### The Turing Way
+This workshop is being run by The Turing Way project team. The Turing Way will be a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". Reproducible research is necessary to ensure that scientific work can be trusted. The goal is to ensure that all results can be independently verified and built upon in future work. Our interdisciplinary team includes researchers, research software engineers and librarians. This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: https://github.com/alan-turing-institute/the-turing-way.
+
+### Programme
+* 9.30 - 10.00: Coffee and Registration
+* 10.00 - 10.30: Introduction to the workshop and The Turing Way
+* 10.30 - 12.00: Reproducible computing environment and how Binder can help
+* 12.00 - 1.00: Lunch
+* 1.00 - 2.00: Zero to Binder
+* 2.00 - 2.30: Coffee
+* 2.30 - 3.30: Build your own Binder
+* 3.30 - 4.00: Demonstrating your Binder, general questions, feedback and close
+* 4.00 - 5.00: Optional hangout with instructors
+
+### Some example Binders:
+*	Office of National Statistics - a report on traffic flow as an early indicator for GDP
+*	A book on digital signal processing, or how computers understand music
+*	A collection of notebooks about neural networks
+*	Example of interactive code snippets in a normal webpage
+
+## Build a BinderHub
+
+Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. Researchers can be supported in doing this through myBinder.org which allows you to share your software with others in the form of a single clickable link, making it easier for users and removing the need for you to worry about supporting lots of different platforms. This is a huge step forward in terms of supporting reproducibility but myBinder.org requires all code and data to be made public, and computational power is limited. One solution to this is to host your own BinderHub locally allowing private code sharing and greater computational power. 
+
+During this free workshop we will discuss how to build your own BinderHub and …. At the end of the workshop you will be able to ….
+
+This workshop is for Research Software Engineers and IT staff who are:
+*	Interested in reproducibility, containers, Docker or continuous integration; 
+*	Already familiar with Binder and R Markdown or Jupyter notebooks;
```

Comment:
Again, not sure about this line. Ultimately, you only really need to know what Binder is. It's the users responsibility to setup their repo/R markdown/Jupyter notebook in such a way that it is Binder-friendly. 🤷‍♀️

Reply:
So are we just asking for familiarity with Binder? Or maybe awareness of Binder? Did you have any thoughts about what people can realistically achieve in the BinderHub workshop? If not I can just leave this out for now 😄

Reply:
I think we're going to demonstrate with Rmarkdown and jupyter so if they don't know either of those then they won't follow the demo. I'd leave it in, I can imagine someone derailing the first part of the workshop with questions about R or python and we can't spend time asking this.

@sgibson91: if you really don't want to mention notebooks, I'd be fine for this to be adjusted to "R Markdown or Python for data science" or something similar, but I do think the participants need to be able to use an R Markdown doc or a jupyter notebook to get started at the beginning of the workshop.